lib/process-services-cloud/src/lib/services/user-preference-cloud.service.ts

```diff
@@ -38,25 +38,30 @@ export class UserPreferenceCloudService extends BaseCloudService implements Pref
    * @param appName Name of the target app
    * @returns List of user preferences
    */
-  getPreferences(appName: string, key?: string): Observable<any> {
+  getPreferences(appName: string): Observable<any> {
     if (appName) {
       const url = `${this.getBasePath(appName)}/preference/v1/preferences`;
-
-      return this.get(url).pipe(map((preferences: any) => {
-        if (key) {
-          const preferencesList = preferences?.list?.entries ?? [];
-          const searchedPreferences = preferencesList.find(preference => preference.entry.key === key);
-          return searchedPreferences ? JSON.parse(searchedPreferences.entry.value) : null;
-        }
-
-        return preferences;
-      }));
+      return this.get(url);
```

Comment:
The method name now is wrong : This is not returning anymore any preferences but only the preference URL